client/guidestours/steps.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react'
+
+/**
+ * Internal dependencies
+ */
+import localize from 'lib/mixins/i18n/localize';
+import Card from 'components/card';
+import Button from 'components/button';
+import ExternalLink from 'components/external-link';
+import Gridicon from 'components/gridicon';
+import { posToCss, bullseyePositioners, getStepPosition } from './positioning';
+
+class GuidesBasicStep extends Component {
+	render() {
+		const stepPos = getStepPosition( this.props );
+		const stepCoords = posToCss( stepPos );
+
+		const { text, onNext, onQuit } = this.props;
+		return (
+			<Card className="guidestours__step" style={ stepCoords } >
+				<p>{ text }</p>
+				<div className="guidestours__buttonrow__choice">
```

Comment:
We need to revise these classes — if all the components will be sub-component of `guides-tour` then we should write them like `guides-tour__{element}`.